java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -1833,6 +1833,10 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_bitwiseMergeAndSetValidit
       default: JNI_THROW_NEW(env, cudf::jni::ILLEGAL_ARG_CLASS, "Unsupported merge operation", 0);
     }
 
+    if (cudf::has_nonempty_nulls(*copy)) {
+      copy = cudf::purge_nonempty_nulls(*copy);
+    }
```

Comment:
Avoid deref `copy` twice: that will create two instances of `column_view`.
```suggestion
    if (auto const copy_cv = copy->view(); cudf::has_nonempty_nulls(copy_cv)) {
      copy = cudf::purge_nonempty_nulls(copy_cv);
    }
```